.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: go
+go:
+- 1.8
```

Comment:
have you tested this against 1.9? Now is the new stable version

Reply:
tested